tests/integration/test_github.py

```diff
@@ -776,11 +789,13 @@ def test_source_project_other_fork_fork(self):
         assert source_project.repo == "hello-world"
 
     def test_source_project_renamed_fork(self):
+        # TODO: why there were "bye-world" in source_project.repo, it needs some manual change?
```

Comment:
As the name of the tests says, we want to check that we get right name even if the upstream and for project name differ. E.g. when you:
- rename the upstream after the fork was forked
- rename the fork

Reply:
yep, that's why I've added there this TODO, because It probably needs some manual change, or something like that and it is not described, what to do, before this test ( for others I skipped these tests.)